frontend-react/src/components/AuthElement.test.tsx

```diff
@@ -197,4 +197,33 @@ describe("AuthElement unit tests", () => {
         const spinner = await screen.findByTestId("rs-spinner");
         expect(spinner).toBeInTheDocument();
     });
+    test("Permits admins whose active membership is not DHPrimeAdmins", () => {
+        mockSessionContext.mockReturnValueOnce({
+            oktaToken: {
+                accessToken: "TOKEN",
+            },
+            memberships: new Map<string, MembershipSettings>().set(
+                "DHPrimeAdmins",
+                {
+                    memberType: MemberType.PRIME_ADMIN,
+                    parsedName: "PrimeAdmins",
+                }
+            ),
+            activeMembership: {
+                memberType: MemberType.NON_STAND,
+                parsedName: "IHaveNoGroupsSadFace",
+            },
+            dispatch: () => {},
+            initialized: true,
+            adminHardCheck: true,
```

Comment:
Do we have a test when adminHardCheck = false?

Reply:
As far as the AuthElement is concerned, it's false in any of the tests above without the `adminHardCheck` value present, but there was something about this that still didn't settle right with me.

I mulled this over a bit, and thought "well, it's a null case that _should_ never happen", so what I did was went and made some SessionContext tests that handle just the logic of that memoized `adminHardCheck` value against 3 cases. Check those out on the updated diff and let me know if the address what the concern was here! :)